src/app/ui/landing-page/landing-page.js

```diff
@@ -41,23 +40,70 @@ let css = csjs`
     width           : 100%;
     font-size       : xx-large;
     align-self      : center;
-
   }
   .text {
     cursor: pointer;
     font-weight: normal;
     max-width: 300px;
     user-select: none;
+    color: var(--primary);
   }
   .text:hover {
-    font-weight: bold;
+    text-decoration: underline;
```

Comment:
I guess first five classes in css not used anymore. Should we remove them?

Reply:
oops just merged after you've approved ;) please feel free to add an issue so we don't forget or directly handle this in a PR..

Reply:
I saw it later so I was late too :). I should rebase after this merge anyway, I'll remove them in my pr if it's ok